### ISSUE TITLE
Reduce Bounces: add a flag per user to disable mail sending

### ISSUE THREAD

**Issue by bugfinder** [openSUSE/open-build-service#11120]
To allow for bounce processing, please add a flag to disabled sending mails for users with a temporarily or permanently broken mailbox

**Comment by hennevogel** [comment]
hm, sounds to me like something you want to do on your MTA no?

**Comment by bugfinder** [comment]
no, the MTAs are the place where the problem occurs now and they just get the bounces back, now we want a way to handle this as you normally do for mailiing lists and such. bouncing mails triggering disabling an account temporarily and if it happens again within shorter timeframe disable permanently

**Comment by lrupp** [comment]
What about including a link to https://build.opensuse.org/my/subscriptions in every Email that is going out of OBS as a first improvement?

A general "disable notifications" option would help us in a 2nd turn to disable those accounts, that have bounces. If this general option would not change the settings on the page above, but just disable EMail notifications and can be easily turned on and off via API call, we are getting very close to what we want.

The current workaround to use a "noreply" address for outgoing Emails from OBS is just a workaround and should not be seen as permanent solution. In normal cases, we want to get replies to Emails.

**Comment by hennevogel** [comment]
> no, the MTAs are the place where the problem occurs now and they just get the bounces back

I mean our sending MTAs, so some [`smtp_recipient_restrictions`](http://www.postfix.org/postconf.5.html#smtpd_recipient_restrictions) rules on our relays.

> What about including a link to https://build.opensuse.org/my/subscriptions in every Email that is going out of OBS as a first improvement?

Sounds like a nice enhancement, please file as a new issue.

> In normal cases, we want to get replies to Emails.

For notification emails? I doubt it. Why would we want to get replies?

**Comment by dmarcoux** [comment]
> > What about including a link to https://build.opensuse.org/my/subscriptions in every Email that is going out of OBS as a first improvement?
> 
> Sounds like a nice enhancement, please file as a new issue.
> 

This isn't needed since it's already in the layout which composes all emails (except specific admin emails). The code below from the emails' layout will add a line at the end of every email with the following text `Configure notifications at https://www.your_domain.com/my/subscriptions`.

https://github.com/openSUSE/open-build-service/blob/fa52c8502944db09a5ef141125dcc7a6e472247a/src/api/app/views/layouts/event_mailer.html.haml#L5
